pymzml/obo.py

```diff
@@ -146,6 +147,15 @@ def __normalize_version(version):
 
         return version
 
+    def download_obo(self, version, obo_file):
+        uri = f"https://raw.githubusercontent.com/pymzml/psi-ms-CV/v{self.version}/psi-ms.obo"
+        urllib.request.urlretrieve(uri, obo_file)
+
+        with open(obo_file, "rb") as fin, gzip.open(obo_file + ".gz", "wb") as fout:
+            fout.writelines(fin.readlines())
```

Comment:
pymzML obo files are psi-ms{self.version}.obo ,... the downloaded is always psi-ms.obo .. will that be a problem?

Reply:
```
obo_file = os.path.join(
            obo_root,
            "obo",
            "psi-ms{0}.obo".format("-" + self.version if self.version else ""),
        )
        if os.path.exists(obo_file):
            pass
        elif os.path.exists(obo_file + ".gz"):
            obo_file = obo_file + ".gz"
        else:
            self.download_obo(self.version, obo_file)
            obo_file += ".gz"
```

The correct name of the obo file is already specified here and the downloaded file is saved under that name